element-{web,desktop}: 1.11.47 -> 1.11.50

yayayayaka opened the pull request:
## Description of changes
- https://github.com/vector-im/element-web/releases/tag/v1.11.50
- https://github.com/vector-im/element-desktop/releases/tag/v1.11.50

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

SuperSandro2000 commented:
might as well update straight to [1.11.49](https://github.com/vector-im/element-web/releases/tag/v1.11.49)

SuperSandro2000 committed:
element-{web,desktop}: 1.11.47 -> 1.11.50

https://github.com/vector-im/element-web/releases/tag/v1.11.50
https://github.com/vector-im/element-desktop/releases/tag/v1.11.50

Co-Authored-By: Yaya <github@uwu.is>

SuperSandro2000 committed:
fetch-yarn-deps: fix missing cert when fetching packages

Found while updating element-web

SuperSandro2000 commented:
Just move the --offline flag to between yarn and run like it is already done for element-desktop

Did also the update to 1.11.50, please pick

https://github.com/SuperSandro2000/nixpkgs/commit/c51e749de1de598baecdc09980728bbaf960b5e7
https://github.com/SuperSandro2000/nixpkgs/commit/4a91e6c30692b9e9fafa1a5c0f177ec1f9e748b9

NickCao reviewed (approved):
Tested element-web.
